### DISCUSSION TITLE
set background color/style of titlebar buttons min, max, close

### DISCUSSION THREAD

**Discussion by automationfuture** [MahApps/MahApps.Metro#4360] [Q&A]
Hello all together,

is there a possibility to change the background color/style of the titlebar buttons min, max, close? I have to do this, but I don't know where to start.

Thanks a lot for your help.

**Comment by punker76** [comment]
@automationfuture Yes, this is possible. Here is a short sample how to do it: https://github.com/punker76/code-samples?tab=readme-ov-file#mahappsmetro-custom-windowbuttoncommands
